app/policies/photo_policy.rb

```diff
@@ -0,0 +1,33 @@
+class PhotoPolicy < ApplicationPolicy
+  attr_reader :user, :photo
+
+  def initialize(user, photo)
+    @user = user
+    @photo = photo
+  end
+
+
+  def show?
+    user == photo.owner ||
+    !photo.owner.private? ||
+    photo.owner.followers.include?(user)
+  end
+
+  def edit?
+    user == photo.owner ||
+    !photo.owner.private? ||
+    photo.owner.followers.include?(user)
+  end
+
+  def update?
+    user == photo.owner ||
+    !photo.owner.private? ||
+    photo.owner.followers.include?(user)
+  end
+
+  def destroy?
+    user == photo.owner ||
+    !photo.owner.private? ||
+    photo.owner.followers.include?(user)
```

Comment:
Do you want the photo owner's followers to be able to delete their photos?

Reply:
Yeah, Ashanti's right - destroy, update, and edit policies should only be `user == photo.owner`